ElectronClient/app/gui/MainScreen.jsx

```diff
@@ -105,6 +123,13 @@ class MainScreenComponent extends React.Component {
 			}
 
 			await createNewNote(null, true);
+		} else if (command.name === 'newTagged') {
+			if (!this.props.folders.length) {
+				bridge().showErrorMessageBox(_('Please create a notebook first'));
+				return;
```

Comment:
There should not be early returns in command handling, because at the end of the function there's some cleanup.

Reply:
I was following the pattern of newNote and newTodo.  (Is that code incorrect?)

Reply:
Oh you're right, I missed that. It's a bug in a previous PR which will have to be fixed. We can't have early returns because it means the command is not cleared at the end, and so it will be triggered again every time the component is rendered.